src/server/utilities/cspHeader/index.test.js

```diff
@@ -536,6 +582,8 @@ describe('cspHeader', () => {
         'https://tpc.googlesyndication.com',
         'https://gn-web-assets.api.bbc.com',
         'https://www.googletagservices.com',
+        'https://securepubads.g.doubleclick.net',
+        'https://bbc.gscontxt.net',
```

Comment:
This seems to be a duplication of line 570?

Reply:
This is for testing: isAmp: false, isLive: false